packages/firebase_ml_vision/android/src/main/java/io/flutter/plugins/firebasemlvision/LocalVisionEdgeDetector.java

```diff
@@ -0,0 +1,104 @@
+package io.flutter.plugins.firebasemlvision;
```

Comment:
So we now have a different structure for the `Detector`s on the native side. We changed them so the user could deallocate resources and have multiple running `Detector`s. You could probably share alot of the logic with the `ImageLabeler.java`.